packages/arrow-data-grid/src/DataGrid.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles'
+
+const HeaderHeightUnits = 7
+
+const CondensetRowHeightUnits = 3
+const DefaultRowHeightUnits = 6.5
+
+interface Heights {
+  header: number
+  row: number
+}
+
+interface UseHeightsProps {
+  rowCondensed: boolean
+}
+
+export const useHeights = ({ rowCondensed }: UseHeightsProps): Heights => {
+  const theme = useTheme()
+
+  return {
+    header: theme.spacing(HeaderHeightUnits),
+    row: theme.spacing(rowCondensed ? CondensetRowHeightUnits : DefaultRowHeightUnits)
+  }
+}
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    '& .MuiDataGrid-windowContainer': {
+      height: 'auto !important'
+    },
+    '& .MuiDataGrid-window': {
+      position: 'inherit',
+      top: '0 !important',
+      marginTop: theme.spacing(7)
+    },
+    '& .MuiDataGrid-viewport': {
+      maxHeight: 'fit-content! important'
```

Comment:
Does `fit-content! important` work? It looks like it should be `fit-content !important`.
(This is repeated several times below)

Reply:
It does. But I just replaced it with the canonical version for consistency.